backend/qualifications/migrations/0001_initial.py

```diff
@@ -75,5 +82,6 @@ class Migration(migrations.Migration):
                 'ordering': ('area', 'name'),
                 'unique_together': {('area', 'name')},
             },
+            bases=(models.Model, common.models.GraphModel),
         ),
     ]
```

Comment:
Rozumim tomu spravne ze jsem te svou no-mercy poznamkou o migracich odbrzdil a pregeneroval jsi je (prip squashnul) i v ostatnich nesouvisejicich appkach?

Pokud ano, tak priste radeji do samostatneho PR prosim -- PR (resp. feature branch na ktere pracujes a kterou posleze publikujes) by mela mit fokus na **jedno** tema, **jeden** problem ktery resi. Nasim tematem je Custom User model, tj. cokoli s touto funkcionalitou nesouvisi musi byt publikovano oddelene.

Duvodem je snadnejsi prace v tymu, primocarejsi code reviews. Pokud vystavim PR na tema A, soustredim se na A a cokoli vybocuje z tematu je podezrele (viz zde -- proc se meni migrace pro dalsi 2 aplikace? jak to souvisi s custom user modelem? unika mi neco?)

Reply:
Nejspíš ti nic neutíká, udělal jsem to přesně tak jak říkáš. Pochopil jsem z předchozího komentáře že to tak mám udělat. Máš ale určitě pravdu, přemýšlím jak to teď ideálně udělat. Když bych vzal staré migrace z ostatních applikací a hodil je sem, a pak commitnul, tak by to mohlo pomoc ne? Nebo myslíš že to není už dál potřeba řešit?

Reply:
Tady to uz dal nijak neresme, mergnu to jak to je. Zkus na to ale priste vic pamatovat: **jeden PR == jeden problem**.